MIGRATE_FROM_VCSIM.md

```diff
@@ -0,0 +1,135 @@
+# Overview
+
+This document explains the current work-in-progress status of migrating from
+[vcsim](https://github.com/vmware/govmomi/tree/master/vcsim). For more
+background on the whole bazel/kind/forklift-process please see the other
+documents in this repo, esp.
+[BUILD_AND_INSTALL_FORKLIFT_WITH_BAZEL_ON_KIND.md](BUILD_AND_INSTALL_FORKLIFT_WITH_BAZEL_ON_KIND.md).
+Later, when everything works reliably, these steps should be integrated into
+one script (or maybe two, one for CI and one for manual execution).
+
+# Step by step instructions
+
+## Build vcsim image without TLS
+
+This needs to be done only once. Get the
+[Dockerfile](https://github.com/vmware/govmomi/blob/master/Dockerfile.vcsim) for vcsim. 
+Change the last line to disable TLS:
+
+    CMD ["-l", "0.0.0.0:8989", "-tls=false"]
+
+Build the image: 
+
+    docker build -f Dockerfile.vcsim -t localhost:5001/vcsim .
```

Comment:
before running this build vcsim binary must downloaded  to the same dir from https://github.com/vmware/govmomi/blob/master/vcsim/README.md  OR built using [make](https://github.com/vmware/govmomi/blob/master/vcsim/Makefile)